### PR TITLE
minor test case for onReady onChange data readOnly and i18n added

### PR DESCRIPTION
**Pull request by robonetphy**

### CODE REVIEW

**Review by talyguryn — APPROVED** [review]

File: test/testcases.md [hunk 1]
```diff
@@ -69,14 +67,42 @@ This document will describe various test cases of the editor.js functionality. F
...
+        - [ ] If `false` passed, the Editor.js should not show the Inline Toolbar for this tool.
+        - [ ] If `array` passed, the Editor.js should show the Inline Toolbar for this tool with a passed list of tools and their order.
+        - [ ] If omitted, the Editor.js should not show the Inline Toolbar for this tool.
       - [ ] Checking the `toolbox` property
         - [ ] If it contains `title`, this title should be used as a tool title
         - [ ] If it contains `icon`, this HTML code (maybe SVG) should be used as a tool icon
+
+- [ ] `onReady` property
+  - [ ] If `function` passed, the Editor.js should call the `function` when it's ready to work.
+  - [ ] If omitted, the Editor.js should be initialized with the `tools` only.
```
Comment (neSpecc) [inline test/testcases.md:80]: I think this line is not useful

File: test/testcases.md [hunk 2]
```diff
@@ -69,14 +67,42 @@ This document will describe various test cases of the editor.js functionality. F
...
+
+- [ ] `onChange` property
+  - [ ] If `function` passed,the Editor.js should call the `function` when something changed in Editor.js DOM.
+  - [ ] If omitted, the Editor.js should be initialized with the `tools` only.
```
Comment (neSpecc) [inline test/testcases.md:84]: and this

File: test/testcases.md [hunk 3]
```diff
@@ -69,14 +67,42 @@ This document will describe various test cases of the editor.js functionality. F
...
+
+- [ ] `data` property
+  - [ ] If omitted
+    - [ ] the Editor.js should be initialized with the `tools` only.
```
Comment (neSpecc) [inline test/testcases.md:88]: same here

File: test/testcases.md [hunk 4]
```diff
@@ -69,14 +67,42 @@ This document will describe various test cases of the editor.js functionality. F
...
+    - [ ] the Editor.js should be empty.
+  - [ ] If `object` passed
+    - [ ] Checking the `blocks` property
+      - [ ] If `array` of `object` passed,
+        - [ ] for each `object` 
+          - [ ] Checking the `type` and `data` property
+            - [ ] the Editor.js should be initialize with `block` of class `type`
+            - [ ] If `type` not present in `tools`, the Editor.js should throw an error.
```
Comment (neSpecc) [inline test/testcases.md:96]: in this case, we use the `Stub` tool. So we need to test it.

Comment (neSpecc) [reply test/testcases.md:96]: Also, we need to check if the `data` property is passing to the tool's `render` method

File: test/testcases.md [hunk 5]
```diff
@@ -69,14 +67,42 @@ This document will describe various test cases of the editor.js functionality. F
...
+      - [ ] If omitted
```
Comment (neSpecc) [inline test/testcases.md:97]: I think this case is already covered above.

File: test/testcases.md [hunk 6]
```diff
@@ -69,14 +67,42 @@ This document will describe various test cases of the editor.js functionality. F
...
+        - [ ] the Editor.js should be initialized with the `tools` only.
+        - [ ] the Editor.js should be empty.
+
+- [ ] `readOnly` property
+  - [ ] If `true` passed,
+    - [ ] If any `tool` have not readOnly getter defined,The Editor.js should throw an error.
+    - [ ] otherwise, the Editor.js should be initialize with readOnly mode. 
+  - [ ] If `false` passed,the Editor.js should be initialized with the `tools` only.
```
Comment (neSpecc) [inline test/testcases.md:105]: can't understand this case

File: test/testcases.md [hunk 7]
```diff
@@ -69,14 +67,42 @@ This document will describe various test cases of the editor.js functionality. F
...
+            - [ ] the Editor.js should be initialize with `block` of class `type`
+            - [ ] If `type` not present in `tools`, the Editor.js should throw an error.
+      - [ ] If omitted
+        - [ ] the Editor.js should be initialized with the `tools` only.
+        - [ ] the Editor.js should be empty.
+
+- [ ] `readOnly` property
+  - [ ] If `true` passed,
+    - [ ] If any `tool` have not readOnly getter defined,The Editor.js should throw an error.
+    - [ ] otherwise, the Editor.js should be initialize with readOnly mode.
```
Comment (neSpecc) [inline test/testcases.md:104]: how to check it?

Comment (robonetphy) [reply test/testcases.md:104]: @neSpecc by checking the `content editable`  property of a block. I guess 🤔

**Comment by robonetphy** [comment]
@neSpecc @talyguryn  @khaydarov 
Please review the test case for i18n.

**Review by neSpecc — APPROVED** [review]

**Pull request merged**